examples/7.8-plugin-postbigextractrequest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Solarium package.
+ *
+ * For the full copyright and license information, please view the COPYING
+ * file that was distributed with this source code.
+ */
```

Comment:
We don't include this in the examples. People would just copy/paste it into their own codebase. 😉